contrib/move-string-resource.kts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * USAGE:
+ * Cuts string resources with the name arg3 from the module from arg1
+ * and pastes them to the string resources from the module from arg2.
+ *
+ * Example usage:
+ * kotlinc -script move-string-resource.kts ../app ../core history_from_current_book
+ *
+ * This will cut all strings with the name history_from_current_book from
+ * all strings.xml files within directories with a prefix of values*
+ * in ../app/src/main/res/ and paste them into
+ * their corresponding files within ../core/src/main/res/values*.
+ *
+ */
+import java.io.File
+import kotlin.system.exitProcess
+
+
+if (args.size < 3) {
+  print_correct_usage_and_exit()
+}
+
+var path_to_values = "/src/main/res/"
```

Comment:
I am not sure if there is kotlin scripting language conventions(please link me if there is) but I would use camelCase and not snake_case

Reply:
Hehe, thought of this just after pushing yesterday. Have done a lot of python as of recently. Will change to camelCase!